README.md

```diff
@@ -1,7 +1,7 @@
 ## Generate live templates
 
 ```
-ruby -Ilib -e "require 'generator'; Generator.dump_all!"
+ruby -Ilib -e "require 'generator'; Generator.dump_all! "
```

Comment:
Kind of a weird hack, but originally was getting a `-bash: !": event not found` error.